packages/gatsby-transformer-video/src/ffmpeg.js

```diff
@@ -0,0 +1,431 @@
+import crypto from "crypto"
+import http from "http"
+import { tmpdir } from "os"
+import { resolve, parse } from "path"
+
+import { pathExists, createWriteStream, ensureDir, unlink } from "fs-extra"
+import ffmpeg from "fluent-ffmpeg"
+import imagemin from "imagemin"
+import imageminGiflossy from "imagemin-giflossy"
+import PQueue from "p-queue"
+
+export default class FFMPEG {
+  constructor({ cacheDir, rootDir, ffmpegPath, ffprobePath }) {
+    this.queue = new PQueue({ concurrency: 1 })
+    this.cacheDir = cacheDir
+    this.rootDir = rootDir
+
+    if (ffmpegPath) {
+      ffmpeg.setFfmpegPath(ffmpegPath)
+    }
+    if (ffprobePath) {
+      ffmpeg.setFfprobePath(ffprobePath)
+    }
+  }
+
+  cacheContentfulVideo = async ({
```

Comment:
shouldn't this be using createRemoteFileNode?

Reply:
done :)